snapshot/git/gitdb/bundlestore.go

```diff
@@ -191,11 +191,13 @@ func (s *bundlestoreSnapshot) Kind() snapshotKind { return s.kind }
 func (s *bundlestoreSnapshot) SHA() string        { return s.sha }
 
 func (s *bundlestoreSnapshot) Download(db *DB) error {
-	log.Print("Return if we already have this sha, else continue by downloading from bundlestore. ", s.SHA())
+	log.Printf("Downloading sha:%s", s.SHA())
```

Comment:
While these changes look clearer, they may not entirely address the original intent of these prints - we want to be very explicit that if an error is printed by the functions in some of these steps, that's ok/expected and the viewer can safely ignore the error.